src/Persistent/Pure.hs

```diff
@@ -512,5 +522,7 @@ mkMetaInfo cUser now oid = MetaInfo
     , _metaChangedAt       = now
     }
 
-nextMetaInfo :: PersistM m => User -> m (MetaInfo a)
-nextMetaInfo cUser = mkMetaInfo cUser <$> getCurrentTimestamp <*> nextId
+nextMetaInfo :: User -> AUpdate (MetaInfo a)
+nextMetaInfo cUser = mkMetaInfo cUser now <$> nextId
+  where
+    now = undefined  -- TODO: #307
```

Comment:
Yes, I was hoping for something like that.  But it could be done later in this PR.